LICENSE

```diff
@@ -200,3 +200,48 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+
+---
+
+
+For the packages/fragment-identifier/src/fragment.js and
+packages/fragment-identifier/src/fragment.pegjs components from
```

Comment:
`fragment.js` is not included in the source release. I think it makes sense to have it in the `.ratignore`, just to avoid confusion for anyone who runs the tool against a dirty project directory, but i doesn't make sense to list here.